staging/src/k8s.io/kubectl/pkg/util/completion/completion.go

```diff
@@ -82,11 +83,11 @@ func ResourceNameCompletionFunc(f cmdutil.Factory, resourceType string) func(*co
 // 1- pod names that match the toComplete prefix
 // 2- resource types containing pods which match the toComplete prefix
 func PodResourceNameCompletionFunc(f cmdutil.Factory) func(*cobra.Command, []string, string) ([]string, cobra.ShellCompDirective) {
-	return func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+	return func(_ *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
```

Comment:
Sorry if this is dumb but what's the pint of this? Is it just because this is possibly being used elsewhere and you just want to throw the first function input away, because it would drastically balloon the PR if you went around and updated the usages everywhere in the code base?

Reply:
The underscore you mean?  It just explicitly indicates that the parameter is unused.  If I recall correctly, the function signature is defined by cobra, so it can't be changed, but we are not using the cmd arg.

Reply:
Oh I see it now, thanks.